src/databricks/labs/remorph/reconcile/query_builder/expression_generator.py

```diff
@@ -3,6 +3,8 @@
 
 from sqlglot import expressions as exp
 
+from databricks.labs.remorph.reconcile.recon_config import Thresholds
+
 
 def _apply_func_expr(expr: exp.Expression, expr_func: Callable, **kwargs) -> exp.Expression:
     level = 0 if isinstance(expr, exp.Column) else 1
```

Comment:
```suggestion
    isTerminal = isinstance(expr, exp.Column)
```

Reply:
Done. linting dosent allow isTerminal. So, changes to is_terminal